dxp/wysiwyg.md

```diff
@@ -0,0 +1,33 @@
+# WYSIWYG Editors
+
+We have recently worked on reducing variability in the WYSIWYG writing experience. After analyzing current usage of the editors the decision was made to provide a more cohesive and comprehensive experience for each specific use case. Documenting the **WYSIWYG** editors that are available in [Liferay DXP](https://github.com/liferay/liferay-portal) can be useful.
```

Comment:
I don't think this introduction is really necessary :)

Reply:
No problem, I'll remove this and stuff comment https://github.com/liferay/liferay-frontend-guidelines/pull/140#discussion_r399190347 when I re-organize the document by the outlines you addressed above.

Reply:
@mateomustapic, the outlines were just an example of things that might fall into this category. As I said, just address this and I can try to come up with an initial draft of what I expect to see here. It should be easier to fill in the blanks than to start from scratch.